components/MainContainer.tsx

```diff
@@ -0,0 +1,106 @@
+import Image from "next/image";
+import React, { useState } from "react";
+import burgerIcon from "./assets/burger.png";
+import closeIcon from "./assets/close.png";
+import styles from "./stylesContainer.module.css";
+import { HEADER_ITEMS } from "../utils/constants";
+import HeaderItem from "./HeaderItem";
+
+const MainWrapper = ({ children }: { children: React.ReactNode }) => {
+  const [sidebarOpen, setSidebarOpen] = useState(false);
+  const [sidebarItemOpen, setSidebarItemOpen] = useState("");
+  return (
+    <>
+      <div className={styles.header + " base_padding"}>
+        <svg
```

Comment:
please extract SVGs to a separate file, no need to add this huge amount of lines to components